include/bouncer.h

```diff
@@ -235,6 +235,7 @@ int pga_cmp_addr(const PgAddr *a, const PgAddr *b);
  * Stats, kept per-pool.
  */
 struct PgStats {
+	uint64_t backend_assignment_count;
```

Comment:
Do we really need this new counter? Isn't `xact_count` effectively the same as `backend_assignment_count`?

Reply:
I guess it's close but not always the same. Transaction count only gets incremented at the end of the transaction and this gets incremented at the start. So yeah this new counter is needed. I do think we should expose this counter to users too though, otherwise it's impossible to calculate the total wait time for users.